### DISCUSSION TITLE
mailparse_msg_parse_file()

### DISCUSSION THREAD

**Discussion by pinkzomo** [php-mime-mail-parser/php-mime-mail-parser#436] [Q&A] [answered]
I get the following error:
`Fatal error: Uncaught Error: Call to undefined function PhpMimeMailParser\mailparse_msg_parse_file() in /var/www/html/public_html/vendor/php-mime-mail-parser/php-mime-mail-parser/src/Parser.php:123`
my code:
```
require_once __DIR__.'/vendor/autoload.php';
$parser = new PhpMimeMailParser\Parser();
$parser -> setPath( $path );
```
It can assign the object (new php...) but then fails (setPath) internally.

Yesterday it worked, today after restart it fails.

composer -vvv about
Running 2.2.6 (2022-02-04 17:00:38) with PHP 8.1.5 on Linux / 5.15.0-100-generic
php -m | grep mailparse
mailparse

**Comment by eXorus** [comment, marked as answer]
For sure mailparse is not installed for the PHP you used.

Maybe it's only activated for PHP CLI and not PHP Web ?

**Reply by pinkzomo** [reply]
Bingo!
Apache was using php 7.4.9 but console was using 8.1. And there was no mailparse for php7.4
After I succeeded in changing php version, voilá it works! 
(There seems to be a bug as a2enmod/a2dismod does not change the links in the directory (etc/.../conf-enabled), so I had to do it manually).
